release-tools/make_bump.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -e
+set -x
+script_dir="$(dirname $(realpath -s $0))"
+
+#tools
+project=$1
+tmp_dir=$(mktemp -t -d cc-bump.XXXX)
+release_tool="${tmp_dir}/release-tool"
+hub_bin="${tmp_dir}/hub-bin"
+
+function cleanup {
+	rm  -rf "${tmp_dir}"
+}
+
+trap cleanup EXIT
+
+die()
```

Comment:
you already have die/info/etc implemented here https://github.com/clearcontainers/packaging/pull/242/files
I wonder if these script should share this functions